### DISCUSSION TITLE
电脑无法访问Youtube——非本教程问题，加PT ipv6相关问题

### DISCUSSION THREAD

**Discussion by DogeCA** [Aethersailor/Custom_OpenClash_Rules#23] [Q&A]
已star，实在是想到头爆也不知道为什么了，Openclash处找了几个回答试过也没什么效果，想咨询下看看大佬知不知道什么情况
网络环境：主软路由拨号，配置Openclash，旁路由仅处理WiFi
Openclash配置情况：按教程走，并配置了ipv6，旁路由默认配置
问题情况：电脑端无论WiFi还是有线都无法正常打开Youtube
Edge完全无法打开
![微信截图_20240813023407](https://github.com/user-attachments/assets/c95fa90e-2f91-4811-b00d-5026b7dc3377)
Chrome处于可打开又不可打开的情况（指首页会出现我账号头像与关注列表），但显示没有联网，如果是点到一些博主主页里就完全不能打开
![微信截图_20240813023653](https://github.com/user-attachments/assets/e2807c03-8056-4bf9-80cb-002ae91923a2)
nslookup是这个情况
![微信截图_20240813023734](https://github.com/user-attachments/assets/26486326-8efe-4f7e-bfc2-d011aaaa50c8)
此前Openclash使用Redir-Host的时候也会出现这问题，偶尔有几次全关浏览器，重启电脑后就可以看了，但现在尝试过也不行
用WiFi的设备除了电脑都是正常的，测试了两台手机和平板都可以打开流畅观看，而电脑连WiFi也是同样情况，所以应该是定位到电脑这边的问题，觉得是大家说的DNS问题，但也不知道如何下手，尝试过Chrome清缓存和winsock reset没作用

另外还有个PT相关，发现v4与v6的流量会全部走漏网之鱼跑流量，现在暂时只能换成直连。
我之前是用插件设置——黑白名单的LAN IP不让QB走代理，但现在没那个选项了，想知道这个规则目前有没有比较方便的PT与代理共存方法，如果没有的话我就自己另加配置吧（主要是这个用起来确实快）。

**Comment by Aethersailor** [comment]
关闭浏览器的安全dns功能
主路由ipv6中关闭和dns一切相关选项

旁路由环境+openclash最好还是舍弃ipv6，问题太多

**Comment by sscjjo** [comment]
主路由，按教程配置好后，检测没有IPV6，IPV4到是无泄露，但就是V6死活没有

**Reply by Aethersailor** [reply]
先关闭openclash试试能否通过国内的ipv6检测网站

**Reply by sscjjo** [reply]
做OC设置之前，电脑是带IPV6的，按OC教程做完后，检测不到IPV6，但在代理中把漏网之鱼改为直通，IPV4无污染，IPV6为国内地址，

**Reply by Aethersailor** [reply]
> 做OC设置之前，电脑是带IPV6的，按OC教程做完后，检测不到IPV6，但在代理中把漏网之鱼改为直通，IPV4无污染，IPV6为国内地址，
> 
> 

你的节点应该是不支持ipv6出站吧，或者节点使用的 dns 不支持返回 AAAA 记录

**Reply by sscjjo** [reply]
https://testipv6.cn/    现在用这个检测能看到IPV4和6是正确的，能上的，但用其它的检测有没有污染，网站是上不去的，另外OC检测下面的网站访问检查四个IP后面的都能上，现在有点迷茫 因为无法看那个https://ipleak.net/是否有泄露 。

**Reply by Aethersailor** [reply]
> https://testipv6.cn/ 现在用这个检测能看到IPV4和6是正确的，能上的，但用其它的检测有没有污染，网站是上不去的，另外OC检测下面的网站访问检查四个IP后面的都能上，现在有点迷茫 因为无法看那个[https://ipleak.net/是否有泄露](https://ipleak.net/%E6%98%AF%E5%90%A6%E6%9C%89%E6%B3%84%E9%9C%B2) 。

https://testipv6.cn/这个是检测国内的ipv6的，你的节点支不支持ipv6？
